go/install/install_osx.go

```diff
@@ -161,6 +161,7 @@ func ListServices(g *libkb.GlobalContext) (*keybase1.ServicesStatus, error) {
 func DefaultLaunchdEnvVars(g *libkb.GlobalContext, label string) []launchd.EnvVar {
 	return []launchd.EnvVar{
 		launchd.NewEnvVar("KEYBASE_LABEL", label),
+		launchd.NewEnvVar("KEYBASE_SERVICE_TYPE", "launchd"), // This was added 03/22/16
```

Comment:
One more question -- in general, why is there a mix of environment variables and command-line flags? To the best of my knowledge, everything in the service should be setting by either/or.  So why not just do all environment variables or all command-line flags?